src/fxa-geodb.js

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I realize I'm being super pedantic here, most of our group's standalone node libraries use the convention that the code is in the `lib` subdir. Consistency makes it easier to understand the source tree.